test/rails/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM brandonmathis/taperole:latest
```

Comment:
I'm a complete docker noob, but does this mean the version of tape that is being tested is your fork? I think we want to do `smashingboxes/taperole`, and somehow, pull in the PR's version of it.